### ISSUE TITLE
2.0.1 Breaks build

### ISSUE THREAD

**Issue by SaraVieira** [simonsmith/postcss-at2x#7]
Hello,

First of all thank you for this plugin but when this was updated to 2.0.1 it broke our build saying:

```
Warning: [object Object] is not a PostCSS plugin Use --force to continue.
```

I'm using grunt to build PostCSS and this is my config:

```
        postcss: {
            options: {
                map: true, // inline sourcemaps

                processors: [
                    require('autoprefixer')({browsers: ['last 3 versions', 'last 4 iOS versions']}),
                    require('postcss-at2x')
                ]
            },
            development: {
                src: 'css/style.css'
            },
            dist: {
                src: 'dist/css/style.css'
            }
        },
``` 

After downgrading to 2.0.0 the build worked fine

**Comment by simonsmith** [comment]
Hey @SaraVieira,

That's odd, very little changed. Could you add the stack trace from grunt?

Thanks

**Comment by SaraVieira** [comment]
Hello,

I noticed there were only small changed that's what struck me as weird.

```
Running "postcss" task

Running "postcss:development" (postcss) task
Verifying property postcss.development exists in config...OK
Files: css/style.css
Options: processors=[null,{}], map, diff=false, safe=false, failOnError=false, writeDest
Warning: [object Object] is not a PostCSS plugin Use --force to continue.
```

I get not stack trace just this even with verbose running

**Comment by simonsmith** [comment]
Fixed in 2.0.2. Looks like I forgot about needing a plugin to handle `module.exports` with Babel. Sorry for breaking the build! :)

**Comment by SaraVieira** [comment]
Updated and it looks great. Thanks man :)
